test/sharness/t0020-init.sh

```diff
@@ -58,11 +58,13 @@ test_expect_success "ipfs config succeeds" '
 	test_cmp expected_config actual_config
 '
 
+test_check_peerid() {
```

Comment:
LGTM

Though this could have been moved to the library, to replace https://github.com/ipfs/go-ipfs/pull/1626/files#diff-21a8cbf17e33b82a8fca825aca160f1dR21